### PR TITLE
throwaway map

### PR DESCRIPTION
**Pull request by jread-usgs**

### CODE REVIEW

File: scripts/fetch/fetch.islands.R [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#' take map arguments and return a projected sp object
+#' 
+#' @param \dots arguments passed to \code{\link[maps]{map}} excluding \code{fill} and \code{plot}
+#' 
+library(maps)
+library(mapdata)
+library(sp)
+to_sp <- function(..., proj.string, within = NULL){
```
Comment (aappling-usgs) [inline scripts/fetch/fetch.islands.R]: is `to_sp` going to get used everywhere? maybe extract it into a helper already?

File: scripts/fetch/fetch.view_limits.R [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                list(proj.string = metadata[['proj.string']],
+                     width = metadata[['width']], 
+                     height = metadata[['height']], 
+                     pointsize = metadata[['pointsize']]))
+  
+  saveRDS(out, viz[['location']])
+}
+
+fetchTimestamp.view_limits <- function(viz){
+  NULL
```
Comment (aappling-usgs) [inline scripts/fetch/fetch.view_limits.R]: this needs to write a file at least once if you don't want it to rebuilt every time. here's one way to do it: https://github.com/USGS-VIZLAB/example/blob/master/scripts/fetch/cars.R#L7

File: scripts/fetch/fetch.islands.R (hunk 1 above)
Comment (jread-usgs) [reply scripts/fetch/fetch.islands.R]: where do we put helper functions?

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,18 @@
+cache/
+target/
+user/
+figures/
+vizlab/
+makefile
```
Comment (jiwalker-usgs) [inline .gitignore]: Makefile? my filesystem is case sensitive

File: scripts/fetch/fetch.islands.R (hunk 1 above)
Comment (jiwalker-usgs) [reply scripts/fetch/fetch.islands.R]: helper functions can be put wherever, but they need to be included in the `scripts` for each thing using them

Comment (aappling-usgs) [reply scripts/fetch/fetch.islands.R]: just another script that's listed in scripts. probably scripts/fetch for this one, so could be scripts/fetch/to_sp.R

**Pull request merged**